apps/crn-frontend/src/discover/__tests__/Routes.test.tsx

```diff
@@ -58,52 +56,78 @@ it('redirects to the guides page when the index page accessed', async () => {
   ).toBeVisible();
 });
 
-it('renders tutorial page when user clicks tutorial card title', async () => {
+it('renders tutorials list page when the tutorials tab is selected', async () => {
   await renderDiscoverPage(discover({}).$);
 
+  mockGetTutorials.mockResolvedValue(createListTutorialsResponse(1));
+
+  const tutorialsAnchorTab = screen.getByText(/Tutorials/i, { selector: 'p' });
+
+  expect(tutorialsAnchorTab).toBeVisible();
+  expect(screen.queryByText(/Explore our tutorials/i)).not.toBeInTheDocument();
+
+  userEvent.click(tutorialsAnchorTab);
+  await waitFor(() =>
+    expect(screen.queryByText(/Loading/i)).not.toBeInTheDocument(),
+  );
+
+  expect(
+    await screen.findByText(/Explore our tutorials/i, { selector: 'p' }),
+  ).toBeVisible();
+});
+
+it('allows search on tutorials list', async () => {
+  mockGetTutorials.mockResolvedValue(createListTutorialsResponse(1));
+
+  await renderDiscoverPage(discover({}).tutorials({}).$);
+
+  userEvent.type(screen.getByRole('searchbox'), 'searchterm');
+
+  await waitFor(() =>
```

Comment:
?

Reply:
Initially I didn't check that the results were displayed because my focus was that entering a search value in the searchbox passed that value in the api request. It doesn't hurt to check though so it's been updated.